org.eclipse.winery.workflowmodeler/package.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
ok, I just add license for code files(ts, css, html). Right now I will add a license header for the config files too.

But I'm not so sure that dose every file must have a license header? I don't know how to add a license header for json files. could you give me some hint ? ths.